packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -435,6 +442,13 @@ class DevFS {
       String archivePath;
       if (deviceUri.path.startsWith(assetBuildDirPrefix))
         archivePath = deviceUri.path.substring(assetBuildDirPrefix.length);
+      // When doing full restart in preview-dart-2 mode, copy content so
```

Comment:
Similarly a TODO here that this clone will not be needed once we switch hot restart to use the IKG

Reply:
Done